src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -710,6 +710,16 @@ TEST_CASE("SimplePaddingLayerTest", "[ANNLayerTest]")
   // Test the Backward function.
   module.Backward(input, output, delta);
   CheckMatrices(delta, input);
+
+  // Test forward function for multiple filters.
+  // Here it's 3 filters with height = 244, width = 244
+  // the output should be 266 * 266 * 3 with 1 padding.
+  Padding<> module1(1, 1, 1, 1, 244, 244);
+  input1 = arma::randu(224 * 224 * 3, 1);
+  module1.Forward(input1, output1);
+  REQUIRE(arma::accu(input1) == arma::accu(output1));
```

Comment:
Looks like this check fails in the pipeline.

Reply:
Yes, it just compiled locally too and i checked the same, i think there are some elements that get lost in the translation but i steped throught it with gdb the dimensions seemed right, can you spot something that might be peoducing this ?

Reply:
So in the code when i assign it to inputTemp it changes for some reason: 
```bash
input: 1.6052e+06
input Temp: 1.57739e+06
```
these were the output of the input and when i was assigned to the `inputTemp` var.